src/main/java/teammates/ui/webapi/GetAccountRequestsAction.java

```diff
@@ -0,0 +1,32 @@
+package teammates.ui.webapi;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import teammates.common.util.Const;
+import teammates.storage.sqlentity.AccountRequest;
+import teammates.ui.output.AccountRequestData;
+import teammates.ui.output.AccountRequestsData;
+
+/**
+ * Action: Gets pending account requests.
+ */
+public class GetAccountRequestsAction extends AdminOnlyAction {
+    @Override
+    public JsonResult execute() {
+        String accountRequestStatus = getNonNullRequestParamValue(Const.ParamsNames.ACCOUNT_REQUEST_STATUS);
+        if (!"pending".equals(accountRequestStatus)) {
```

Comment:
Maybe we should put `"pending"` in a static class in `Const`, but I am not sure if that's weird because it would only have one value, unlike `Const.EntityType`. Another way is to just have it be a static variable in `GetAccountRequestsAction`, but I don't think this is also done in the other `Action` classes. I lean more towards the former than the latter.

Reply:
Would it be better to directly use `AccountRequestStatus` and do a case-insensitive match. This way we don't have to create any new variables.
Something like this:
```java
const pendingValue = AccountRequestStatus.PENDING.name();  // 'PENDING'
if (!pendingValue.equals(accountRequestStatus.toUpperCase())) {
```

Reply:
Using `AccountRequestStatus` sounds good so that we don't need to redefine anything